doc/bind.md

```diff
@@ -5,26 +5,12 @@
 
 @signature `value.bind( object, keyPath )`
 
-In the example below, a `keyObservable` is created that is two-way bound to the
```

Comment:
probably should be a `KeyObservable` and we should probably eventually create docs for it.

Reply:
I'm assuming that this should be an issue?